docs/how_to/tensorflow_install/tensorflow_install.md

```diff
@@ -9,6 +9,19 @@ training and inference in neural networks. It is one of the most popular and
 in-demand frameworks and is very active in open source contribution and
 development.
 
+:::{warning}
+ROCm 5.6 and 5.7 deviate from the standard practice of supporting the last three
+TensorFlow versions. This is due to incompatibilities between earlier TensorFlow
+versions and changes introduced in the ROCm 5.6 compiler. Please refer to the
+following version support matrix:
+
+| ROCm  | TensorFlow |
+|:-----:|:----------:|
+| 5.6.x | 2.12       |
+| 5.7.0 | 2.12, 2.13 |
+| Post-5.7.0 | Back to standard support practice. |
```

Comment:
What is 'standard support practice'? How can users find this information? Can we add a link to best practices?

Reply:
I have made the description more explicit

Reply:
Thanks so much, Nara.